modules/using-an-algorithm/pages/index.adoc

```diff
@@ -12,11 +12,138 @@ Each algorithm has its own folder, where you can find the implementation of the
 
 GDS Library algorithms are implemented in GSQL as GSQL queries.
 Therefore, the process to run an algorithm query is the same as the process to run any other query.
-The names of GDS algorithm queries as provided by the library begin with the prefix `tg_` only to distinguish them from a user-written query.
-However, since the library is open-source, you can change the name of the queries to any name of your choosing.
 
+The algorithms come pre-installed with TigerGraph version 3.8.0 and later as packaged queries.
+On earlier versions of TigerGraph before 3.8.0, and in xref:gui:graphstudio:overview.adoc[GraphStudio], each query must be installed manually.
 
-== Check for data or schema constraints
+Most of the algorithm queries provide multiple output formats.
+You usually have the option to output one or more of the following:
+
+* JSON output to console
+* CSV output to a file
+* Results written to an attribute of vertices or edges
+
+== Packaged template queries
+
+Packaged template queries are a new, simplified method for working with query algorithms that are preinstalled in a TigerGraph instance.
+They use a new keyword, `CALL`, to run with just-in-time compilation for improved performance.
+
+The syntax is very similar to using a package in Python.
+
+=== Import and explore template queries
+
+Using a TigerGraph installation version 3.8.0 and later, run the following command to import the query package.
```

Comment:
In TigerGraph version 3.8.0 and later, run the following command to improve the query package.